src/main/views/macro/internalLink.njk

```diff
@@ -0,0 +1,3 @@
+{% macro internalLink(text, href) %}
```

Comment:
I would recommend removing this, we should be using html if it the macro doesn't give any benefit
the externalLink one gives a benefit that the opens in a new windows is always applied and applied consistently.
This just hides what's actually happen for no benefit I can see

Reply:
I added it reads better and is cleaner to do:

{% set privacyPolicyUri = AppPaths.privacyPolicyPage.uri %}
  <p>
        {{ t('This includes the ' + internalLink('privacy policy', privacyPolicyUri)) | safe }}
      </p>

Then to do 

  <p> {{ t('This includes the <a href="{{ uri }}">privacy policy</a>.',
        { uri: AppPaths.privacyPolicyPage.uri }) | safe }}
      </p>

Also, if were to have a macro for externalLinks, then it would make sense and be consistent that we have one for internal links. 
Its faster to utilise and is very quick to change an internal link to external link as it holds the same structure.

Reply:
there's no structure to it and it would never change? 
It gives value when doing interpolation I see